usr/local/www/diag_resetstate.php

```diff
@@ -43,21 +43,31 @@
 ##|*MATCH=diag_resetstate.php*
 ##|-PRIV
 
+define("DEBUG", true);	// Displays "success" messages, but does not call the functions that reset the firewall state tables
```

Comment:
Yes and no :)

I send you a PR, you make a suggestion and I experiment with that suggestion. I still need to test the page layout and appearance so I still need those DEBUG data arrays.

If you prefer you could just tell me that the PR is ready to be merged and I could remove the DEBUG, but of course if it needs to be revised in the future, we will be stuck again.


> On May 8, 2015, at 9:54 AM, SjonHortensius <notifications@github.com> wrote:
> 
> In usr/local/www/diag_resetstate.php <https://github.com/SjonHortensius/pfsense/pull/154#discussion_r29940942>:
> 
> > @@ -43,21 +43,31 @@
> >  ##|*MATCH=diag_resetstate.php*
> >  ##|-PRIV
> >  
> > +define("DEBUG", true);	// Displays "success" messages, but does not call the functions that reset the firewall state tables
> I understand the intention, but I'm not sure it should be part of this branch :)
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/SjonHortensius/pfsense/pull/154/files#r29940942>.
> 

Steve
--
Stephen Beaver | +1 (512) 646-4100;239 | sbeaver@netgate.com <mailto:sbeaver@netgate.com> | www.netgate.com <http://www.netgate.com/>